node/Cargo.toml

```diff
@@ -3,67 +3,82 @@ authors = ["Eliott Teissonniere <git.eliott@teissonniere.org>"]
 build = "build.rs"
 edition = "2018"
 name = "nodle-chain"
-version = "2.0.13"
+version = "2.0.12"
```

Comment:
why are we lowering the version?

Reply:
Thanks, @ETeissonniere, it's my bad. Restored it back :+1: